Refresh.GameServer/Types/Playlists/LevelPlaylistRelation.cs

```diff
@@ -0,0 +1,19 @@
+using Realms;
+using Refresh.GameServer.Types.Levels;
+
+namespace Refresh.GameServer.Types.Playlists;
+
+/// <summary>
+/// A mapping of playlist -> sub-level
+/// </summary>
+public partial class LevelPlaylistRelation : IRealmObject
+{
+    /// <summary>
+    /// The playlist the level is contained in
+    /// </summary>
+        public int PlaylistId { get; set; }
```

Comment:
Formatting. Also, we point to `GameLevel` by object but point to `GamePlaylist` by ID in this schema? Is this some weird Postgres preparation thing?
```suggestion
    public int PlaylistId { get; set; }
```

Reply:
I don't remember why I did it like this, there's no reason to in this case, I'll move it to be by-reference